src/components/main/Main.js

```diff
@@ -1,12 +1,38 @@
 import React from 'react';
+
```

Comment:
we made webpack setup to avoid importing common modules; please, do not use `import React from 'react';` and use [ProvidePlugin](https://webpack.js.org/plugins/provide-plugin/)